lib/hawkular/operations/operations_api.rb

```diff
@@ -134,7 +134,7 @@ def invoke_specific_operation(operation_payload, operation_name, &callback)
     #
     # @param callback [Block] callback that is run after the operation is done
     def add_deployment(hash, &callback)
-      hash[:enabled] ||= true
+      hash[:enabled] = defined?(hash[:enabled]) ? hash[:enabled] : true
```

Comment:
The result is the same, but indeed it seems more adequate to use the specific hash method `key?` instead of the generic `defined?`. Will change.